packages/explat-client-react-helpers/src/index.tsx

```diff
@@ -56,6 +60,22 @@ export default function createExPlatClientReactHelpers(
 			}
 
 			return [ isLoading, experimentAssignment ];
-		},
+		};
+
+	const Experiment = ( { children, name: experimentName }: { children: { control: React.ReactNode, treatment: React.ReactNode, loading: React.ReactNode }, name: string } ): JSX.Element => {
+		const [isLoading, experimentAssignment] = useExperiment(experimentName)
+		if (isLoading) {
+			return <>{children.loading}</>
+		} else if (experimentAssignment?.variationName === 'treatment') {
+			return <>{children.treatment}</>
+		} else {
+			return <>{children.control}</>
+		}
```

Comment:
In line with our suggestions in other clients, it'd be better not to rely on the variation name and reverse the condition. Something like this:

```suggestion
		} else if (!experimentAssignment?.variationName) {
			return <>{children.control}</>
		} else {
			return <>{children.treatment}</>
		}
```

Also, it may be better to keep the terminology consistent and replace `control` with `default`.

Reply:
Oh we do currently return `null` for `isDefault=true`?

Reply:
Yeah, unless there's a bug somewhere. `Assigned_Variation::to_assigner_return_value()` should be called from all assigner return points.

```php
class Assigned_Variation {
	...

	public static function to_assigner_return_value( ?Assigned_Variation $assigned_variation ): ?string {
		if ( ! $assigned_variation->variation || $assigned_variation->variation->is_default ) {
			return null;
		}
		return $assigned_variation->variation->name;
	}
}
```